network-api/networkapi/buyersguide/factory.py

```diff
@@ -66,7 +46,7 @@ def product_category(self, create, extracted, **kwargs):
         """
         ceiling = 1.0
         while True:
-            odds = get_random_float()
+            odds = random()
             if odds < ceiling:
                 category = get_random_category()
```

Comment:
```suggestion
                category = choice(BuyersGuideProductCategory.objects.all())
```

That way, you can remove ` def get_random_category`. Don't forget to import choice ;)